internal/orchestrator/updates.go

```diff
@@ -318,10 +321,14 @@ func conditionFromUpdateEvent(evt *v1types.ConditionUpdateEvent) *rctypes.Condit
 	}
 }
 
-func filterIncompleteRecords(records []*store.ConditionRecord) []*store.ConditionRecord {
+func (o *Orchestrator) filterIncompleteRecords(records []*store.ConditionRecord) []*store.ConditionRecord {
```

Comment:
I'd rather not make this a member function just to avoid passing a parameter.

Reply:
thanks, fixed now